src/main/java/org/jabref/gui/externalfiletype/CustomizeExternalFileTypesDialog.java

```diff
@@ -40,6 +40,7 @@ public CustomizeExternalFileTypesDialog() {
         this.setResultConverter(button -> {
             if (button == ButtonType.OK) {
                 viewModel.storeSettings();
+                fileTypesTable.refresh();
```

Comment:
If it does not work, then you can delete this indeed.

Reply:
done